index.js

```diff
@@ -12,7 +12,7 @@ const junk = require('junk');
 const readFile = promisify(fs.readFile);
 const writeFile = promisify(fs.writeFile);
 
-const handleFile = async (sourcePath, {destination, plugins = []}) => {
+const handleFile = async (sourcePath, base, {destination, preserveDirectories, plugins = []}) => {
```

Comment:
If a user has a glob like this `assets/images/**/*.{jpg,svg,webp}` we need the base in order to copy all the files in the destination without the `assets/images/` directory/